packages/mdjs-core/types/remark.d.ts

```diff
@@ -27,11 +27,3 @@ declare module 'rehype-autolink-headings' {
 
   export = unified.Plugin;
 }
-
-declare module 'unist-util-remove' {
-  import unified from 'unified';
-
-  function remove(ast: unified.Node, opts: any, test?: any): unified.Node;
-
-  export = remove;
-}
```

Comment:
I think I had a type error caused by this so I removed this, perhaps we don't need this file anymore, pretty sure remark has decent types nowadays?